library/core/src/num/f64.rs

```diff
@@ -1024,29 +1024,37 @@ impl f64 {
         Self::from_bits(u64::from_ne_bytes(bytes))
     }
 
-    /// Returns an ordering between self and other values.
+    /// Return the ordering between `self` and `other`.
+    ///
     /// Unlike the standard partial comparison between floating point numbers,
     /// this comparison always produces an ordering in accordance to
-    /// the totalOrder predicate as defined in IEEE 754 (2008 revision)
-    /// floating point standard. The values are ordered in following order:
-    /// - Negative quiet NaN
-    /// - Negative signaling NaN
-    /// - Negative infinity
-    /// - Negative numbers
-    /// - Negative subnormal numbers
-    /// - Negative zero
-    /// - Positive zero
-    /// - Positive subnormal numbers
-    /// - Positive numbers
-    /// - Positive infinity
-    /// - Positive signaling NaN
-    /// - Positive quiet NaN
-    ///
-    /// Note that this function does not always agree with the [`PartialOrd`]
-    /// and [`PartialEq`] implementations of `f64`. In particular, they regard
-    /// negative and positive zero as equal, while `total_cmp` doesn't.
+    /// the `totalOrder` predicate as defined in the IEEE 754 (2008 revision)
+    /// floating point standard. The values are ordered in the following sequence:
+    ///
+    /// - Negative quiet NaN;
+    /// - Negative signaling NaN;
+    /// - Negative infinity;
+    /// - Negative numbers;
+    /// - Negative subnormal numbers;
+    /// - Negative zero;
+    /// - Positive zero;
+    /// - Positive subnormal numbers;
+    /// - Positive numbers;
+    /// - Positive infinity;
+    /// - Positive signaling NaN;
+    /// - Positive quiet NaN.
+    ///
+    /// The ordering established by this function does not always agree with the
+    /// [`PartialOrd`] and [`PartialEq`] implementations of `f64`. For example,
+    /// they consider negative and positive zero equal, while `total_cmp`
+    /// doesn't.
+    ///
+    /// The iterpretation of signalling NaN bit follows the IEEE 754 standard,
+    /// which may not match the interpretation by some of the older,
+    /// non-conformant hardware implementations.
```

Comment:
I would specifically state the currently known example for MIPS since out of context it's unclear how someone should interpret this message.